build.gradle

```diff
@@ -98,17 +113,32 @@ project('ripc-transport-netty4') {
 
 		// Netty
 		compile "io.netty:netty-all:$nettyVersion"
+        compile "org.slf4j:slf4j-api:1.7.6"
 	}
 }
 
-project('ripc-composition-rxjava1') {
+project('ripc-rxjava1') {
```

Comment:
The reason to remove "composition" was to not indicate that it is just for functional composition. In the other issue, we are discussing to rename the layer to "adapter" (or adaptor), so when we do that we can make the name change part of the project.